### PR TITLE
fix a regression introduced by switching to the new JES API

### PR DESCRIPTION
**Pull request by scottfrazer**
- [x] Run full Tyburn tests to make sure remaining failing workflows now pass

This appears to NOT affect the GotC workflow, but it does affect 3 of the Tyburn WDLs

### CODE REVIEW

File: src/main/scala/cromwell/engine/backend/jes/JesBackend.scala [hunk 1]
```diff
@@ -92,7 +92,7 @@ object JesBackend {
    * @return A path which is unique per input path
    */
   def localFilePathFromCloudStoragePath(gcsPath: GcsPath): Path = {
-    Paths.get(JesWorkingDisk.MountPoint).resolve(gcsPath.bucket).resolve(gcsPath.objectName)
+    Paths.get(gcsPath.bucket).resolve(gcsPath.objectName)
```
Comment (scottfrazer) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: JES works with relative paths now (relative to the disk they're localized to), not absolute paths

**Comment by mcovarr** [comment]
:+1:

**Comment by scottfrazer** [comment]
Don't merge yet,  I just discovered another small issue which should take care of the last tyburn failure

**Comment by scottfrazer** [comment]
@Horneth and @mcovarr can you review when you get a chance?  This should now fix the Tyburn failures on the `develop` branch

File: src/main/scala/cromwell/engine/backend/jes/JesBackend.scala [hunk 2]
```diff
@@ -433,26 +432,29 @@ case class JesBackend(actorSystem: ActorSystem)
...
+      case y if y.getAttribute("name").sourceString.startsWith("write_") => y
+    })
+
+    val evaluatedExpressionMap = writeFunctionAsts map { ast =>
+      val expression = WdlExpression(ast)
+      val value = expression.evaluate(backendCall.lookupFunction(Map.empty), backendCall.engineFunctions)
+      expression.toWdlString.md5SumShort -> value
+    } toMap
+
+    val writeFunctionFiles = evaluatedExpressionMap collect { case (k, v: Success[_]) => k -> v.get } collect { case (k, v: WdlFile) => k -> Seq(v)}
```
Comment (mcovarr) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: can this be one `collect`?

**Comment by mcovarr** [comment]
:+1: again :smile:

File: src/main/scala/cromwell/engine/backend/jes/JesBackend.scala (hunk 2 above)
Comment (scottfrazer) [inline src/main/scala/cromwell/engine/backend/jes/JesBackend.scala]: I did try this:

```scala
evaluatedExpressionMap collect { case (k, v: Success[WdlFile]) => k -> Seq(v.get) }
```

However, I got a type erasure warning from the compiler, which is why I made it two collects.

**Comment by Horneth** [comment]
:+1:

**Comment by scottfrazer** [comment]
I'll merge this soon... I think I can increase test coverage a bit first

**Pull request merged**